Source/WebCore/animation/CSSPropertyAnimation.cpp

```diff
@@ -2342,6 +2341,22 @@ class PropertyWrapperSVGPaint final : public AnimationPropertyWrapperBase {
     void (RenderStyle::*m_setter)(const StyleColor&);
 };
 
+class PropertyWrapperFontWeight final : public PropertyWrapper<FontSelectionValue> {
+    WTF_MAKE_FAST_ALLOCATED;
+public:
+    PropertyWrapperFontWeight()
+        : PropertyWrapper<FontSelectionValue>(CSSPropertyFontWeight, &RenderStyle::fontWeight, &RenderStyle::setFontWeight)
+    {
+    }
+
+private:
+
+    void blend(RenderStyle& destination, const RenderStyle& from, const RenderStyle& to, const CSSPropertyBlendingContext& context) const final
+    {
+        (destination.*m_setter)(FontSelectionValue(std::min(std::max(1.0f, blendFunc(static_cast<float>(this->value(from)), static_cast<float>(this->value(to)), context)), 1000.0f)));
```

Comment:
What is the `1000` value coming from? Might it be more appropriate to be a named constant?